.github/workflows/tox.yml

```diff
@@ -36,24 +36,45 @@ jobs:
           python-version: "3.x"
       - name: run packaging
         run: |
-          python -m pip install -U tox
-          tox -e packaging
+          python3 -m pip install -U tox
+          python3 -m tox -e packaging
   test:
+    name: ${{ matrix.name }}
     needs: lint
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        python: ["3.6", "3.7", "3.8", "3.9"]
+        include:
+          - name: py36
+            tox_env: py36
+            os: ubuntu-20.04
+            python-version: 3.6
+          - name: py37
+            tox_env: py37
+            os: ubuntu-20.04
+            python-version: 3.7
+          - name: py38
+            tox_env: py38
+            os: ubuntu-20.04
+            python-version: 3.8
+          - name: py39
+            tox_env: py39
+            os: ubuntu-20.04
+            python-version: 3.9
+          - name: py38@macos
+            tox_env: py38
+            os: macOS-latest
+            python-version: 3.8
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@master
         with:
           python-version: ${{ matrix.python }}
       - name: run test
         run: |
-          python -m pip install tox
-          tox -e py$(printf "${{ matrix.python }}" | tr -d '.')
+          python3 -m pip install tox
+          python3 -m tox -e py$(printf "${{ matrix.python }}" | tr -d '.')
```

Comment:
```suggestion
          python3 -m tox -e "${{ matrix.tox_env }}"
```

Reply:
An alternative would be to eliminate the tox_env value from it and use the `${{ matrix.python-version }}` value now.